src/Model.php

```diff
@@ -900,6 +902,8 @@ public function save($data = [])
             return $this;
         }
 
+        $this->_dirty_after_save = [];
```

Comment:
yeah it was initially caled dirty_after_save and i renamed it into dirty_after reload, let me check if this is necessary.